packages/frontend/src/components/AudioRecorder/Encoder.ts

```diff
@@ -0,0 +1,106 @@
+import { Mp3Encoder } from '@breezystack/lamejs'
+
+interface EncoderConfig {
+  sampleRate?: number
+  bitRate?: number
+}
+
+class Encoder {
```

Comment:
You mentioned that this is a forked version of https://github.com/closeio/mic-recorder-to-mp3 ? We need to comply with its license. Putting a top-level JSDoc comment in each file with `@license` and the text of the original license notice should work.  
And in general I think this should be made clear that this is just copy-pasted, so it should be treated as a dependency, until we actually separate it out into a dependency.

Reply:
This should be sufficient?